src/LanguageServer/Impl/Implementation/Server.WorkspaceSymbols.cs

```diff
@@ -152,7 +158,11 @@ await GetModuleVariablesAsync(entry as ProjectEntry, opts, @params.query, 50, ca
                 res.children = Array.Empty<DocumentSymbol>();
             }
 
-            var loc = m.Locations.FirstOrDefault(l => !string.IsNullOrEmpty(l.FilePath));
+            var loc = m.Locations
```

Comment:
The only implementation of `IMemberResult` is a struct `MemberResult`. We should either use that struct directly (without interface) or make it a class. 
Also, `Locations` can't be null, and I'd actually expect `IEnumerable<T>` properties and methods to never return null. `MaybeEnumerate` is an anti-pattern.

Reply:
Actually they do can be null. It may be poor design and we can try and clean it up, but that's how it is today... For the sake of not crashing, just following current code patterns.

Reply:
Take it back, not in the member result.